.github/actions/list-slices/action.yml

```
@@ -26,6 +29,6 @@ runs:
       id: list-slices
       shell: ${{inputs.shell}}
       run: |
-        "$TACOS_GHA_HOME/"lib/ci/list-slices <<'EOF'
+        "$TACOS_GHA_HOME/"lib/ci/list-slices ${{inputs.allowlist}} <<'EOF'
```

If you instead pass this via an environment variable, it avoids the possibility of weird characters making the bash parser confused.

```suggestion
  env:
    allowlist: ${{inputs.allowlist}}
  run: |
        "$TACOS_GHA_HOME/"lib/ci/list-slices "$allowlist" <<'EOF'
```

It probably doesn't matter here, but it's quite important elsewhere so it's a good habit.